FLT/ForMathlib/MeasurableSpacePadics.lean

```diff
@@ -0,0 +1,50 @@
+import Mathlib.MeasureTheory.Measure.Haar.Basic
```

Comment:
I was planning on moving away from `ForMathlib` and adopting Yael's `Mathlib` approach. Can we move this into the `Mathlib` directory?

Reply:
Yep!